### ISSUE TITLE
shuffle dataloader?

### ISSUE THREAD

**Issue by kayburns** [MarkFzp/act-plus-plus#50]
https://github.com/MarkFzp/act-plus-plus/blob/26bab0789d05b7496bacef04f5c6b2541a4403b5/utils.py#L268C64-L268C83

By default the torch DataLoader class does not shuffle: https://pytorch.org/docs/stable/data.html

Should DataLoader be modified to take shuffle=True?
